src/analytics.py

```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+from raw import request_analytics
+
+
+def filter_raw_analytics(raw_analytics):
+    # Reset index and rename tiers
+    df = (raw_analytics.reset_index()
+                       .rename(columns={'index': 'raw'}))
+
+    # Split raw column into multiple columns
+    columns = ['card_id', 'archetype_id', 'tier']
+    df[columns] = df.raw.str.split('.', expand=True)
+
+    # Record only consolidated data by archetype and change index to titleId
+    df = (df[df.archetype_id == 'ALL'].drop(['raw', 'archetype_id'], axis=1)
+                                      .set_index('card_id'))
+
+    # Replace abbreviations with tier full names
+    df.replace({'b': 'Bronze',
+                's': 'Silver',
+                'g': 'Gold',
+                'p': 'Platinum'},
+               inplace=True)
+
+    # Unnest statistics
+    unnest = ['games', 'wins', 'check', 'copies']
+    df[unnest] = pd.DataFrame(df.explode([0])[0].to_list(),
+                              index=df.index).iloc[:, :4]
+
+    # Set index to integer
+    df.index = df.index.astype('int64', copy=False)
+
+    return df[['tier', 'wins', 'copies']]
+
+
+def get_analytics_games(df):
+    "Returns data frame 'analytics_games'"
```

Comment:
Normally we use triple `"` to docstring.

Reply:
Fair enough!